lib/Eisbach_Methods.thy

```diff
@@ -549,4 +549,7 @@ method solves_emerging methods m1 m2 = (rule bool_protectD, (m1 ; (rule bool_pro
 
 end
 
+method repeat_unless methods stop cont
```

Comment:
This is cool :)

Reply:
Thanks! Though credit to Matt for the actual implementation

Reply:
:+1:

Reply:
And credit to @Xaphiosis for the original idea.